applications/HelloIOIO/build.gradle

```diff
@@ -6,6 +6,9 @@ android {
     compileSdkVersion 30
     defaultConfig {
 
+        // Default version necessary for some reason.
```

Comment:
"some reason" ! 
Either delete it, or explain it

Reply:
I would love to. Still working on finding the actual reason. Were you able to reproduce with the steps I detailed? If not, I'll focus on local possibilities. 

A more suitable fix seems to be moving these different minSdkVersions to different build types. Are you opposed to that? I'll make those changes instead if that seems like a viable option for the testing you have implemented.

Reply:
I'm able to reproduce !

An additional way is to go in general with sample apps with `minSdk 21`

Reply:
Thats true. My Nexus 5 is running 21, so I wouldn't personally have a need for anything less, but maybe others would. I'm for moving to 21 and creating a separate build type if anyone creates and issue requesting back down to 14. 

I tried a [manifest] fix, but was still getting build errors. It seems most people point to flavors or build types for this though.